.gitignore

```diff
@@ -1 +1,2 @@
 .tmp
+node_modules
```

Comment:
Did you remove this for any particular reason? I can't get it running without it. 

Let me know your thoughts if you did? Would like to know